theme/templates/resource-landing-page/quota.html

```diff
@@ -1,54 +1,59 @@
 {% load hydroshare_tags %}
-{% if is_owner_user %}
-    <div class="panel panel-default">
-        <div class="panel-heading">
-            <h3 class="panel-title">Storage</h3>
-        </div>
-        <div class="panel-body">
-            <span>The quota holder of this resource is:</span>
-            <span>
-                {% include "includes/profile-link.html" with user=quota_holder %}
-                {% include "includes/profile-card.html" %}
-            </span>
+<div v-if="selfAccessLevel === 'owner'" class="panel panel-default">
+    <div class="panel-heading">
+        <h3 class="panel-title">Storage</h3>
+    </div>
+    <div class="panel-body">
+        <span>The quota holder of this resource is: </span>
+        <span><a :href="'/user/' + quotaHolder.id">${quotaHolder.best_name}</a></span>
+        {# TODO: create a reusable component for this template #}
+{#        <span>#}
```

Comment:
clean up comments...

Reply:
I see, it's just a TODO so it's fine as-is

Reply:
Yes, I'll leave it in place until the next PR with the reusable component is merged. This is addressed there.